modules/rtticketinfo.php

```diff
@@ -43,7 +43,7 @@
 //$SESSION->save('backto', $_SERVER['QUERY_STRING']);
 
 $ticket = $LMS->GetTicketContents($id);
-
+$ticket['relatedtickets'] = $LMS->GetRelatedTicketIds($id);
```

Comment:
Czy tego nie będziemy zawsze robić przy ładowaniu ticketu? Jeśli tak by było to może wpakować ten kod do GetTicketContents()? Czy niekoniecznie?

Reply:
Nie koniecznie bo w przypadku rtticketinfoshort nie są potrzebne te informacje. To wypełnia panel powiązanymi ticketami.

Reply:
Dobry przykład!